src/segments.zsh

```diff
@@ -3,7 +3,7 @@
 
 # Defualts
 __BLOX_SEG_DEFAULT__UPPER_LEFT=( host cwd git exec_time )
-__BLOX_SEG_DEFAULT__UPPER_RIGHT=( bgjobs nodejs pyenv virtualenv time )
+__BLOX_SEG_DEFAULT__UPPER_RIGHT=( bgjobs nodejs pyenv virtualenv awsprofile )
```

Comment:
```suggestion
__BLOX_SEG_DEFAULT__UPPER_RIGHT=( bgjobs nodejs pyenv virtualenv awsprofile time )
```

Why override the `time` block? I put it between the `virtualenv` and the `time` blocks.